src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/DiskReplicationConfiguration/AzureToAzureDiskReplicationConfiguration.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ----------------------------------------------------------------------------------
+// 
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
+{
+    /// <summary>
+    ///     Creates Azure Site Recovery Disk replication configuration for A2A replication.
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.New,
+        "AzureToAzureDiskReplicationConfiguration",
+        DefaultParameterSetName = ASRParameterSets.AzureToAzure,
+        SupportsShouldProcess = true)]
+    [OutputType(typeof(ASRAzureToAzureDiskReplicationConfiguration))]
+    public class ASRAzureToAzureDiskReplicationConfiguration : SiteRecoveryCmdletBase
+    {
+        #region Parameters
+
+        /// <summary>
+        ///     Gets or sets the disk uri.
+        /// </summary>
+        [Parameter(ParameterSetName = ASRParameterSets.AzureToAzure, Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string VhdUri { get; set; }
+
+        /// <summary>
+        ///     Gets or sets the primary staging/ log storage account ARM Id.
+        /// </summary>
+        [Parameter(ParameterSetName = ASRParameterSets.AzureToAzure, Mandatory = true)]
```

Comment:
Correct me if I am wrong..
LogStrorageAccount  and RecoveryStorageAccount both can be fetched from Find /Get-Resource command and passed as Pipe. If both value take input from pipeline it will not able to understand which property to Set.
As per my understanding only one property can be taken as ValueFromPipelineByPropertyName in parameter Set.